src/encoded/tests/test_batch_download.py

```diff
@@ -135,6 +135,39 @@ def test_batch_download_lookup_column_value(lookup_column_value_item, lookup_col
g19\t\t\t\treleased\t\t\tcharacterization(s) pending review, missing derived_from, mismatched status, missing raw data in replicate, experiment not submitted to GEO\tmissing possible_controls\tinconsistent target\nENCFF959VGP\tfastq\treads\tENCSR706IDL\tDNase-seq\tUBERON:0001891\tmidbrain\ttissue\tHomo sapiens\t\t\t\t\t\t\t\t\t\t\tDNA\t\t\t\t\tFalse\t\tRoadmap\t\t\t\t1\t1\t36\t\tsingle-ended\t\t\t\t471059102\tJohn Stamatoyannopoulos, UW\tc9585d7a5c4d13d5a12cc0d2b6a4b05c\t\thttp://localhost/files/ENCFF959VGP/@@download/ENCFF959VGP.fastq.gz\t\t\tIllumina HiSeq 2000\t\treleased\t\t\tmismatched status\tmissing documents\t\nENCFF477MLC\tfastq\treads\tENCSR751YPU\tDNase-seq\tUBERON:0001891\tmidbrain\ttissue\tHomo sapiens\t\t\t\t\t\t\t\t\t\t\tDNA\t\t\t\t\tFalse\t2015-08-31\tRoadmap\t\t\t\t1\t1\t36\t\tsingle-ended\t\t\t\t2130374031\tJohn Stamatoyannopoulos, UW\t99c57d24b11f839feea4824565f4ff1a\t\thttp://localhost/files/ENCFF477MLC/@@download/ENCFF477MLC.fastq.gz\t\t\tHiSeq 2500\t\treleased\t\t\texperiment not submitted to GEO, mismatched status\tmissing documents, unreplicated experiment\t\nENCFF000LBC\tcsqual\treads\tENCSR751YPU\tDNase-seq\tUBERON:0001891\tmidbrain\ttissue\tHomo sapiens\t\t\t\t\t\t\t\t\t\t\tDNA\t\t\t\t\tFalse\t2015-08-31\tRoadmap\t\t\t\t1\t1\t50\t\t\t\t\t\t4706779893\tMichael Snyder, Stanford\t233fb8f21fc7c975ab742ca38caeaad5\t\thttp://localhost/files/ENCFF000LBC/@@download/ENCFF000LBC.csqual.gz\t\t\tApplied Biosystems SOLiD System 3 Plus\t\treleased\t\t\texperiment not submitted to GEO, mismatched status\tmissing documents, unreplicated experiment\t\nENCFF000LBB\tcsqual\treads\tENCSR751YPU\tDNase-seq\tUBERON:0001891\tmidbrain\ttissue\tHomo sapiens\t\t\t\t\t\t\t\t\t\t\tDNA\t\t\t\t\tFalse\t2015-08-31\tRoadmap\t\t\t\t1\t1\t50\t\tpaired-ended\t2\tENCFF000LBC\t\t4763145717\tMichael Snyder, Stanford\tfd942e78195f509cf0454ec7621eaa7a\t\thttp://localhost/files/ENCFF000LBB/@@download/ENCFF000LBB.csqual.gz\t\t\tApplied Biosystems SOLiD System 3 Plus\t\treleased\t\t\texperiment not submitted to GEO, mismatched status\tmissing documents, unreplicated experiment\t\nENCFF000LBA\tcsfasta\treads\tENCSR751YPU\tDNase-seq\tUBERON:0001891\tmidbrain\ttissue\tHomo sapiens\t\t\t\t\t\t\t\t\t\t\tDNA\t\t\t\t\tFalse\t2015-08-31\tRoadmap\t\t\t\t1\t1\t25\t\tpaired-ended\t1\tENCFF000LAZ\t\t2307361344\tMichael Snyder, Stanford\t51046aaf12d18c9c314d524f4599c98f\t\thttp://localhost/files/ENCFF000LBA/@@download/ENCFF000LBA.csfasta.gz\t\t\tApplied Biosystems SOLiD System 3 Plus\t\treleased\t\t\texperiment not submitted to GEO, mismatched status\tmissing documents, unreplicated experiment\t\nENCFF000LAZ\tcsfasta\treads\tENCSR751YPU\tDNase-seq\tUBERON:0001891\tmidbrain\ttissue\tHomo sapiens\t\t\t\t\t\t\t\t\t\t\tDNA\t\t\t\t\tFalse\t2015-08-31\tRoadmap\t\t\t\t1\t1\t25\t\tpaired-ended\t2\tENCFF000LBA\t\t2314662239\tMichael Snyder, Stanford\t20125f316dafa4daed390afae59e7bb2\t\thttp://localhost/files/ENCFF000LAZ/@@download/ENCFF000LAZ.csfasta.gz\t\t\tApplied Biosystems SOLiD S
```

Comment:
the expected value is thorough but very hard to change.    Like you could test the first line last line and number of lines or something.

In fact this will break if the number of Experiments in the test set changes.

Reply:
Before I do that, how does this seem to you- Unit test supposed to be brittle. If not, they may not catch subtle changes that ought to be caught. That logic is fine to me, but is it fine to the rest of the world? :)